submarine-commons/commons-unixusersync/src/test/java/org/apache/submarine/commons/unixusersync/EmbeddedLdapRuleTest.java

```diff
@@ -133,6 +136,39 @@ public void testList() throws Exception {
     context.close();
   }
 
+  @Test
+  public void testauth() throws Exception {
+    DirContext ctx = null;
+    Hashtable<String, String> HashEnv = new Hashtable<String, String>();
+
+    String loginId = "uid=curie,dc=example,dc=com";
+    String password = "password";
+
+    HashEnv.put(Context.SECURITY_AUTHENTICATION, "simple");
+    HashEnv.put(Context.SECURITY_PRINCIPAL, loginId);
+    HashEnv.put(Context.SECURITY_CREDENTIALS, password);
+    HashEnv.put(Context.INITIAL_CONTEXT_FACTORY, "com.sun.jndi.ldap.LdapCtxFactory");
+    HashEnv.put("com.sun.jndi.ldap.connect.timeout", "3000");
+    HashEnv.put(Context.PROVIDER_URL, "ldap://ldap.forumsys.com:389");
+
+    try {
+      ctx = new InitialDirContext(HashEnv);
+      LOG.info("Pass");
+    }
+    catch (AuthenticationException e) {
+      LOG.info("fail");
```

Comment:
Please delete `LOG.info(fail)`
Because there is already have `LOG.error(e.getMessage(), e);`